llm_bioinformatics_research/server.js

```diff
@@ -135,6 +135,37 @@ app.post('/profile', async (req, res) => {
     }
 });
 
+app.post('/home', async (req, res) => {
+    const { userId, theme, language } = req.body;
+
+    try {
+        const database = client.db("user_information");
+        const collection = database.collection("user_credentials");
+
+        const result = await collection.updateOne(
+            { _id: new ObjectId(userId) },
+            {
+                $set: {
+                    profile_photo,
+                    phone_number,
+                    location,
+                    theme: theme,
+                    language: language
+                }
+            }
+        );
+
+        if (result.modifiedCount === 0) {
+            return res.status(400).json({ message: "No changes made or user not found." });
+        }
+
+        res.status(200).json({ message: "Settings updated successfully" });
```

Comment:
I noticed that you are currently displaying a message when the setting have been successfully updated. While I like the idea of notifying the user with a message, maybe we can make that more subtle, like a small pop-up in the lower right corner off the screen that disappears after 10 seconds? Or something along those lines.

<img width="481" alt="image" src="https://github.com/user-attachments/assets/3ee25e0d-813b-463a-9d3b-da13f1a3c414">